backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dividedcard/BpkDividedCard.kt

```diff
@@ -39,13 +40,15 @@ fun BpkDividedCard(
   secondaryContent: @Composable () -> Unit,
   modifier: Modifier = Modifier,
   onClick: (() -> Unit)? = null,
+  isElevated: BpkCardElevation = BpkCardElevation.Default,
```

Comment:
rather than `isElevated` can you name this `elevation` to be consistent with the rest of the component please?